samples/HealthChecksSample/LivenessProbeStartup.cs

```diff
@@ -17,7 +15,7 @@ public void ConfigureServices(IServiceCollection services)
             // Registers required services for health checks
             services
                 .AddHealthChecks()
-                .AddCheck(new SlowDependencyHealthCheck());
+                .AddCheck<SlowDependencyHealthCheck>("Slow", failureStatus: null, tags: new[] { "ready", });
```

Comment:
We could consider reordering some parameters to make these signatures better for optional parameters. What isn't great is the ones that take a delegate, because its natural to put the delegate at the end. Thoughts?